dbms/src/AggregateFunctions/AggregateFunctionStatisticsSimple.h

```diff
@@ -139,8 +136,180 @@ struct VarMomentsDecimal
             throw Exception("Decimal math overflow", ErrorCodes::DECIMAL_OVERFLOW);
         return convertFromDecimal<DataTypeDecimal<T>, DataTypeNumber<Float64>>(tmp / (m0 - 1), scale);
     }
+};
+
+template <typename T, size_t _level>
+struct HighOrderMoments
+{
+    T m0{};
+    T m1{};
+    T m2{};
+    T m3{};
+    T m4{};
```

Comment:
It's wrong that we serialize / deserialize / keep in memory value even if it is unneeded.